lib/score_tracker.rb

```diff
@@ -48,6 +50,7 @@ def self.is_a_column_full(board, character)
 
   def self.extract_columns(board)
     @@board = board
```

Comment:
why does board need to be assigned to the class variable @@? It look you don't even need the @@board variable